### ISSUE TITLE
Make tests more modular. split data prep from analysis.

### ISSUE THREAD

**Issue by aiqc** [aiqc/AIQC#67]
## Background
`tests.py` has tests for each: 
- data type (tabular, image)
- analysis type (binary classify, multi label classify, regression)
- library (pytorch, keras)

The `make_test_batch_*` function runs all of the low level api calls needed to prepare data to test that type of analysis.

## Problem
When we add tensorflow, there will be 3 different libraries all making duplicate data preparation steps for `make_test_batch_*`

## Solution
Make a functions that prepare low level api objects for:
- tabular classify binary
- tabular classify multi
- tabular classify regression
- image classify binary

So that they can be used by either keras, pytorch, or tensorflow
